spec/services/versioned_files_service/versions_manifest_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe VersionedFilesService::VersionsManifest do
+  subject(:manifest) { described_class.new(path: pathname) }
+
+  let(:pathname) { Pathname.new('tmp/versions_manifest.json') }
+
+  after do
+    pathname.delete if pathname.exist?
+  end
+
+  describe '.update_version' do
+    context 'when backfilling withdrawn versions' do
+      let(:now) { DateTime.now }
+
+      let(:expected_manifest) do
+        {
+          "$schemaVersion" => 1,
+          'head' => 3,
+          'versions' => {
+            '1' => { 'state' => 'permanently_withdrawn' },
```

Comment:
Purl is expecting no underscore https://github.com/sul-dlss/purl/blob/1a60b041a89359b61ee9b9c0133ee2b5a983342f/spec/fixtures/document_cache/wp/335/yr/5649/wp335yr5649/versions/versions.json#L3

Reply:
Can we change purl? DSA also uses permanently_withdrawn and it would be nice to be consistent across systems.

Reply:
Also, the original ticket specifies `permanently_withdrawn`: https://github.com/sul-dlss/purl/issues/1101